controllers/ContactUsController.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Request\ContactUsRequest;
+// use Illuminate\Http\Request;
+use Illuminate\Http\RedirectResponse;
+use Illuminate\Contracts\View\View;
+use Illuminate\Mail\Message;
+
+class ContactUsController extends Controller
+{
+    public function view(){
+        return view('contacts.contactUs');
+    }
+
+    public function send(ContactUsRequest $request): RedirectResponse
+    {
+        // dd($request->validated());
+
+        $callback = function (string $test): string {
+             return "{$test} input";
+         };
+
+
+        $data = $request->validated();
+
+        \Log::debug($callback('test'), $data);
+
+        \Mail::send(
+            'emails.contactUs', 
+            [
+                'email' => $data['email'],
+                'name' => $data['name'],
+                'districts' => $data['districts'],
+                'department' => $data['department'],
+                'messageText' => $data['message'],
+           
+            ],
+            function (Message $message) use ($data) {
+                $message->subject('message from ' . $data['email']);
+                $message->to('tech@baloon.app');
```

Comment:
please adjust to and from values to your project baloon word doesn't look good.

Reply:
change email[ ](https://github.com/liubamiron/bootcamp-project/pull/9/commits/2cd09b59b361372a4c592e4d0b7b781f0bafd8a1)